src/SceneManager.ts

```diff
@@ -305,6 +306,49 @@ export class SceneManager {
     );
     this.transport.subscribe(poseTopic);
 
+    // Map of audio URIs to Audio objects
+    let audioMap = {};
+    // Subscribe to the audio control topic.
+    const audioTopic: Topic = {
```

Comment:
I was planning on making audio control it's own class rather than directly integrating into the SceneManager. Can you give that a shot?

Reply:
Sure, will take a look.

Reply:
Take a look a 5ed3b6a, where I make a new GzAudioTopic class and use that instead.